src/client/views/layouts/play.ejs

```diff
@@ -27,4 +27,5 @@
         </main>
         <%- include(`${viewsfolder}/components/footer`, {t:t, languages:languages, language:language}) %>
     </body>
+    <script src="/scripts/memberHeader.js"></script>
```

Comment:
Can this be moved to in the head by the other scripts? It feels of of place to be at the bottom of the document